packages/react-hig/src/adapters/GlobalNav/GlobalNavAdapter.js

```diff
@@ -0,0 +1,192 @@
+/**
+Copyright 2016 Autodesk,Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+import * as HIG from 'hig.web';
+import * as PropTypes from 'prop-types';
+
+import HIGElement from '../../elements/HIGElement';
+import HIGChildValidator from '../../elements/HIGChildValidator';
+import createComponent from '../createComponent';
+
+import SideNavComponent, { SideNav } from '../../elements/components/GlobalNav/SideNav';
+import TopNavComponent, { TopNav } from '../../elements/components/GlobalNav/TopNav/TopNav';
+import SubNavComponent, { SubNav } from '../../elements/components/GlobalNav/SubNav';
+import Slot from '../../elements/components/GlobalNav/Slot';
+
+class GlobalNav extends HIGElement {
+  constructor(initialProps) {
+    super(HIG.GlobalNav, initialProps);
+
+    ['_toggleSideNav', '_render'].forEach(fn => {
+      this[fn] = this[fn].bind(this);
+    });
+
+    this.state = {
+      sideNavOpen: false
+    };
+  }
+
+  componentDidMount() {
+    // Add any children
+    if (this.sideNav) {
+      this.hig.addSideNav(this.sideNav.hig);
+      this.sideNav.mount();
+    }
+
+    if (this.topNav) {
+      this.hig.addTopNav(this.topNav.hig);
+      this.topNav.mount();
+      this._addSideNavOpenCallback(this.topNav);
+    }
+
+    if (this.subNav) {
+      this.hig.addSubNav(this.subNav.hig);
+      this.subNav.mount();
+    }
+
+    if (this.initialProps.sideNavOpen) {
+      this.hig.showSideNav();
+    } else {
+      this.hig.hideSideNav();
+    }
+
+    if (this.slot) {
+      this.hig.addSlot(this.slot);
+    }
+
+    this._render();
```

Comment:
I don't see this defined anywhere, and I don't think our adapters will ever need `_render`. Eliminate with extreme prejudice.